### DISCUSSION TITLE
Keep windows size

### DISCUSSION THREAD

**Discussion by vanstrouble** [rxhanson/RectanglePro-Community#290] [Ideas]
Hello! First of all, I am really enjoying the Rectangle Pro experience. Great job!
The idea that I propose is that when making a Custom Shortcut we could choose something like "Keep size when moving the window". Usually when we resize the windows and later move them, they return to their previous state. Later, if we want the window to keep that size we need to close and reopen it. I don't know if that's a MacOS feature or if it can be done with Rectangle Pro, but that would be nice.
![Captura de pantalla 2023-06-07 a la(s) 10 28 02 a m](https://github.com/rxhanson/RectanglePro-Community/assets/89953802/e6dd891c-ef7e-49ac-b50a-82543242ee50)

**Comment by rxhanson** [comment]
Thanks for the kind feedback!

If you don't want the "unsnap restore", there is a checkbox for that in the Settings tab.

If you just want a custom shortcut that moves the window but does not resize it, then you can leave the W & H dimensions blank and just set a docking position.

Let me know if that's not what you're looking for.

**Reply by vanstrouble** [reply]
Right, my mistake! Thank you very much for the feedback.
However, it would be nice if it was an option in the Custom Shortcuts so it doesn't affect how it works on the predefined Snaps. I think if the option is added in Custom Shortcuts it increases customization. It's just my opinion. Thanks for everything.
